packages/tablecloth-base/tablecloth-base.0.0.8/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Native OCaml library implementing Tablecloth, a cross-platform standard library for OCaml and Rescript"
+description: """
+Tablecloth is an ergonomic, cross-platform, standard library for use with OCaml and Rescript. It provides an easy-to-use, comprehensive and performant standard library, that has the same API on in OCaml and Rescript.
+"""
+maintainer: "Paul Biggar <paul@darklang.com>"
+authors: [
+  "Paul Biggar <paul@darklang.com>"
+  "Dean Merchant <deanmerchant@gmail.com>"
+  "Pomin Wu <pomin.wu@proton.me>"
+]
+license: "MIT with some exceptions"
+homepage: "hhttps://github.com/darklang/tablecloth-ocaml-base"
```

Comment:
Drive-by-comment: 
```difff
- homepage: "hhttps://github.com/darklang/tablecloth-ocaml-base"
+ homepage: "https://github.com/darklang/tablecloth-ocaml-base"
```

Same typo also appears in https://github.com/ocaml/opam-repository/pull/25024

Reply:
Many thanks.  Closing this and created https://github.com/ocaml/opam-repository/pull/25032 that includes other fixes.